test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -2123,7 +2123,7 @@ public void EntryPointMulticlassPipelineEnsemble()
             }
         }
 
-        [LessThanNetCore30OrNotNetCoreFact("netcoreapp3.0 output differs from Baseline")]
+        [LessThanNetCore30OrNotNetCoreFact("netcoreapp3.1 output differs from Baseline")]
```

Comment:
>netcoreapp3.1 output differs from Baseline" [](start = 44, length = 43)

Have you tried netcore3.1 also generates different result?

Reply:
Hi @frank-dong-ms , good point, I'm now checking for this in one of my draft testing PRs.

Reply:
In [this build](https://dev.azure.com/dnceng/public/_build/results?buildId=566312&view=logs&j=9d6f93fc-7103-540d-abb9-b79bad552b0d&t=ef9fb040-bca3-54d1-40bf-7394d88ead2f) where I ran these changes without the `LessThanNetCore30OrNotNetCoreFact`, we can see that the benchmarks are indeed different than the baselines.